keystorage/aws/src/main/java/tech/pegasys/signers/aws/AwsSecretsManager.java

```diff
@@ -48,9 +48,8 @@ public static AwsSecretsManager createAwsSecretsManager(
     return new AwsSecretsManager(secretsManagerClient);
   }
 
-  public static AwsSecretsManager createAwsSecretsManager(final String region) {
-    final SecretsManagerClient secretsManagerClient =
-        SecretsManagerClient.builder().region(Region.of(region)).build();
+  public static AwsSecretsManager createAwsSecretsManager() {
```

Comment:
Can you reduce the scope of this and the other create method to package level so that only CacheableAwsSecretsManagerProvider can create the AwsSecretsManager